Description: Use user home to save data and private confs.
Author: Matias De lellis <mati86dl@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- kuntur-2.2.orig/lib/dtv-util/src/fs.cpp
+++ kuntur-2.2/lib/dtv-util/src/fs.cpp
@@ -150,11 +150,13 @@ std::string make( const std::string &p1,
 }
 
 REGISTER_INIT_CONFIG( fs ) {
+	boost::filesystem::path path( util::homeDirectory() );
+	path /= ".local/kuntur";
 	root().addNode("fs")
 		.addValue("user", "Is user tool", UTIL_USE_USER_PREFIX )
 		.addValue("install", "Install root directory prefix", UTIL_INSTALL_PREFIX )
-		.addValue("conf", "Configuration root directory prefix", UTIL_CONFIG_PREFIX )
-		.addValue("data", "Data root directory prefix", UTIL_DATA_PREFIX );
+		.addValue("conf", "Configuration root directory prefix", path.string() )
+		.addValue("data", "Data root directory prefix", path.string() );
 }
 
 const std::string installRootDir() {
